oculus_ros2/src/sonar_viewer.cpp

```diff
@@ -15,54 +15,54 @@ SonarViewer::SonarViewer(rclcpp::Node* node) : node_(node) {
 
 SonarViewer::~SonarViewer() {}
 
-// void SonarViewer::stream_and_filter(const int &width,
+// void SonarViewer::streamAndFilter(const int &width,
```

Comment:
I don't know if we should leave the filtering code inside this code. I think the filtering should occur somewhere else.

Reply:
In an other C++ class in oculus_ros2 package or in an other repo ?